data/chameleoncloud/sites/uc/clusters/chameleon/nodes/303310ee-baf0-4bd3-b3c9-7f28d14936e4.json

```diff
@@ -5,111 +5,96 @@
     "gpu_vendor": "NVIDIA"
   },
   "main_memory": {
     "humanized_ram_size": "192 GiB",
-    "ram_size": 205084688384
+    "ram_size": 206158430208
   },
   "monitoring": {
     "wattmeter": false
   },
   "network_adapters": [
     {
-      "bridged": false,
       "device": "eno1",
       "driver": "i40e",
       "enabled": true,
       "interface": "Ethernet",
       "mac": "24:6e:96:7d:a5:1e",
-      "management": false,
-      "model": "Ethernet Controller X710 for 10GbE SFP+ (rev 01)",
-      "mounted": true,
+      "model": "Ethernet Controller X710 for 10GbE SFP+",
       "rate": 10000000000,
-      "vendor": "Intel Corporation"
+      "vendor": "Intel"
     },
     {
-      "bridged": false,
       "device": "eno2",
       "driver": "i40e",
-      "enabled": false,
+      "enabled": true,
       "interface": "Ethernet",
       "mac": "24:6e:96:7d:a5:20",
-      "management": false,
-      "model": "Ethernet Controller X710 for 10GbE SFP+ (rev 01)",
-      "mounted": false,
+      "model": "Ethernet Controller X710 for 10GbE SFP+",
       "rate": 10000000000,
-      "vendor": "Intel Corporation"
+      "vendor": "Intel"
     },
     {
-      "bridged": false,
       "device": "eno3",
       "driver": "igb",
       "enabled": false,
       "interface": "Ethernet",
       "mac": "24:6e:96:7d:a5:3e",
-      "management": false,
-      "model": "Intel Corporation I350 Gigabit Network Connection (rev 01)",
-      "mounted": false,
+      "model": "I350 Gigabit Network Connection",
       "rate": 1000000000,
-      "vendor": "Intel Corporation"
+      "vendor": "Intel"
     },
     {
-      "bridged": false,
       "device": "eno4",
       "driver": "igb",
       "enabled": false,
       "interface": "Ethernet",
       "mac": "24:6e:96:7d:a5:3f",
-      "management": false,
-      "model": "Intel Corporation I350 Gigabit Network Connection (rev 01)",
-      "mounted": false,
+      "model": "I350 Gigabit Network Connection",
       "rate": 1000000000,
-      "vendor": "Intel Corporation"
+      "vendor": "Intel"
     }
   ],
   "node_name": "nc28",
   "node_type": "gpu_rtx_6000",
   "placement": {
-    "node": 28,
-    "rack": 3
+    "node": "28",
+    "rack": "3"
   },
   "processor": {
-    "cache_l1": null,
     "cache_l1d": 32768,
     "cache_l1i": 32768,
     "cache_l2": 1048576,
     "cache_l3": 20185088,
     "clock_speed": 2600000000,
-    "instruction_set": "x86-64",
-    "model": "Intel Xeon",
-    "other_description": "Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz",
-    "vendor": "Intel",
-    "version": null
+    "instruction_set": "x86_64",
+    "model": "Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz",
+    "vendor": "Intel"
   },
   "storage_devices": [
     {
       "device": "sda",
-      "driver": "megaraid_sas",
       "humanized_size": "240 GB",
-      "interface": "SATA",
+      "interface": "SAS",
+      "media_type": "SSD",
       "model": "MZ7KM240HMHQ0D3",
+      "serial": "S3BSNX0J800366",
```

Comment:
the serials are useful for noting when a part was replaced, but generally not useful to have as a selector in the hardware catalog.